Description: Compatibility patch to support compilation of tomcat7
 using openjdk-7
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/modules/jdbc-pool/src/main/java/org/apache/tomcat/jdbc/pool/DataSource.java
+++ b/modules/jdbc-pool/src/main/java/org/apache/tomcat/jdbc/pool/DataSource.java
@@ -28,6 +28,8 @@
 import org.apache.juli.logging.Log;
 import org.apache.juli.logging.LogFactory;
 
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.logging.Logger;
 
 /**
  * A DataSource that can be instantiated through IoC and implements the DataSource interface
@@ -157,5 +159,7 @@
         }
     }
 
-
+    public Logger getParentLogger() throws SQLFeatureNotSupportedException {
+        throw new SQLFeatureNotSupportedException();
+    }
 }
